packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -1733,6 +1868,14 @@ All groups have the following properties:
 
 > **Note:** The `id`, `created`, `lastUpdated`, `lastMembershipUpdated`, `objectClass`, `type`, and `_links` properties are available only after you create a Group.
 
+<ApiLifecycle access="ea" />
```

Comment:
I'm wondering if we should move this to be next to `source` in the first column of the table. On the page, it seems like it could be a little ambiguous to some people whether it applies to the group attributes in the table above or not:

<img width="576" alt="Screen Shot 2020-09-20 at 6 15 13 PM" src="https://user-images.githubusercontent.com/28817198/93726921-73bd1500-fb6d-11ea-922d-cfbe0a5f6753.png">